python/semantic_kernel/kernel_extensions/kernel_filters_extension.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from functools import partial
+from typing import Any, Callable, Coroutine, Literal, TypeVar
+
+from pydantic import Field
+
+from semantic_kernel.filters.filter_context_base import FilterContextBase
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+FILTER_CONTEXT_TYPE = TypeVar("FILTER_CONTEXT_TYPE", bound=FilterContextBase)
+CALLABLE_FILTER_TYPE = Callable[[FILTER_CONTEXT_TYPE, Callable[[FILTER_CONTEXT_TYPE], None]], None]
+
+
+ALLOWED_FILTERS = ["function_invocation", "prompt_rendering", "auto_function_invocation"]
```

Comment:
Ah, I see them defined here. As a developer how can I easily know what the allowed filters are? Are they only present in our current samples? I am assuming our public docs will cover them, too, but not everyone reads those.